chain/chunks/src/lib.rs

```diff
@@ -2020,6 +2020,27 @@ impl ShardsManager {
             ShardsManagerRequestFromClient::CheckIncompleteChunks(prev_block_hash) => {
                 self.check_incomplete_chunks(&prev_block_hash)
             }
+            ShardsManagerRequestFromClient::ProcessOrRequestChunk {
+                candidate_chunk,
+                request_header,
+                prev_hash,
+            } => {
+                if let Err(e) = self.process_partial_encoded_chunk(candidate_chunk.into()) {
+                    warn!(target: "chunks", "Error processing partial encoded chunk: {:?}", e);
```

Comment:
nit: do not interpolate error, just add it as a separate field: `warn!(target: "chunks", ?err, "Error processing partial encoded chunk");`

Reply:
Addressed as part of https://github.com/near/nearcore/pull/10480/commits/2a4d18189d23cc4a4a8d72b248ac65b8d87006e0